packages/content-element/package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "@tailor-cms/tce-custom-element",
+  "version": "0.0.1",
+  "description": "Content element main package for edit and display templates",
+  "author": "ExtensionEngine <info@extensionengine.com>",
+  "type": "module",
+  "main": "index.ts"
```

Comment:
Should be a bundled version => https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html#including-declarations-in-your-npm-package